x-pack/plugins/security_solution/public/common/components/event_details/overview/index.tsx

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { find } from 'lodash/fp';
+import type { Severity } from '@kbn/securitysolution-io-ts-alerting-types';
+
+import type { BrowserFields } from '../../../containers/source';
+import { euiStyled } from '../../../../../../../../src/plugins/kibana_react/common';
+import type { TimelineEventsDetailsItem } from '../../../../../common/search_strategy/timeline';
+import type { EnrichedFieldInfo, EnrichedFieldInfoWithValues } from '../types';
+import { getEnrichedFieldInfo } from '../helpers';
+import {
+  ALERTS_HEADERS_RISK_SCORE,
+  ALERTS_HEADERS_RULE,
+  ALERTS_HEADERS_SEVERITY,
+  SIGNAL_STATUS,
+} from '../../../../detections/components/alerts_table/translations';
+import {
+  SIGNAL_RULE_NAME_FIELD_NAME,
+  SIGNAL_STATUS_FIELD_NAME,
+} from '../../../../timelines/components/timeline/body/renderers/constants';
+import { FormattedFieldValue } from '../../../../timelines/components/timeline/body/renderers/formatted_field';
+import { OverviewCardWithActions } from '../overview/overview_card';
+import { StatusPopoverButton } from '../overview/status_popover_button';
+import { SeverityBadge } from '../../../../../public/detections/components/rules/severity_badge';
+import { useThrottledResizeObserver } from '../../utils';
+
+export const NotGrowingFlexGroup = euiStyled(EuiFlexGroup)`
+  flex-grow: 0;
```

Comment:
I think there is a `grow` prop on the flex group or maybe the flex item.

Reply:
I tried to add `grow={false}` to the `EuiFlexItems` that are rendered by this but it did not do the trick as the parent (the `EuiFlexGroup`) was still expanding to full height. Probably because it's nested in another FlexGroup from outside?